### DISCUSSION TITLE
Add github & slack integration

### DISCUSSION THREAD

**Discussion by vladciobancai** [helmfile/helmfile#1308] [Ideas]
Hello!

Is it possible to add "github & slack integration"(https://github.com/integrations/slack) app to helmfile/helmfile repo?

Link to setup https://github.com/apps/slack/installations/new

Thank you.

**Comment by yxxhero** [comment]
more details. please.

**Comment by vladciobancai** [comment]
no need it anymore, I used feed app from slack to receive notification when a release is made
